### PR TITLE
Deploy on npm package

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,4 +35,7 @@ Other docs:
   developers.
 * [Application development guidelines](docs/guidelines.md) – This guide is for
   application developers using ngeo.
-* [Transifex](https://www.transifex.com/camptocamp/ngeo/) – Translation platform
+* [npm](https://www.npmjs.com/package/ngeo) – Package on npm.
+* [Transifex](https://www.transifex.com/camptocamp/ngeo/) – Translation platform.
+* [Travis](https://travis-ci.org/camptocamp/ngeo) – Continus integration.
```
Comment (tsauerwein) [inline README.md]: Continuous integration.

Comment (sbrunner) [inline README.md]: Thanks, fixed :-)

**Comment by tsauerwein** [comment]
LGTM

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**